README.md

```diff
@@ -444,6 +444,91 @@ openshift.withCluster( 'mycluster' ) {
 }
 ```
 
+However, be forewarned, controllers or other users can update objects while you are trying to 
+update objects, and you can get collision conflicts.  Consider:
+
+1. Pruning any object `status` or other optional fields you are not concerned with
+2. If pruning `java.util.Map` object returned from `object()` on selectors, you'll
+need to make sure the `remove` method is in the pipeline groovy method whitelist.
+3. When pruning, you have to make sure required fields have at least a minimal or empty
+setting, even if you are not overriding them
+
+As examples, first, here is a version where we get the selector for a Deployment Config
+we created from and `openshift.newApp` call, and then pruned various fields to avoid
+conflicts, and then updated an expected port's name:
+
+```groovy
+openshift.withCluster() {
+    openshift.withProject() {
+        def app = openshift.newApp('registry.access.redhat.com/jboss-fuse-6/fis-java-openshift')
+        def dc = app.narrow('dc')
```

Comment:
another option might be to re-get the dc via "oc export" which strips most of this stuff for you.